### PR TITLE
AMPI: Add attribute AMPI_MY_HOME_WTH

### PR DESCRIPTION
**Pull request by evan-charmworks**

### CODE REVIEW

**Review by stwhite91 — DISMISSED** [review]

File: doc/ampi/04-extensions.rst [hunk 1]
```diff
@@ -477,7 +477,8 @@ threads, processes, and hosts in the job. We define a worker thread to
 be a thread on which one of more AMPI ranks are scheduled. We define a
 process here as an operating system process, which may contain one or
 more worker threads. The built-in attributes are ``AMPI_MY_WTH``,
```
Comment (matthiasdiener) [inline doc/ampi/04-extensions.rst:479]:
```suggestion
more worker threads. The built-in attributes are listed in the following table:

+------------------------+-------------------------------------------------+
| Variable               | Defintion                                       |
+========================+=================================================+
| ``AMPI_MY_WTH``        |                                                 |
|                        | Worker thread the rank is currently running on. |
+------------------------+-------------------------------------------------+
| ``AMPI_MY_PROCESS``    | OS process the rank is currently running on.    |
+------------------------+-------------------------------------------------+
| ``AMPI_NUM_WTHS``      | Number of worker threads in the application.    |
+------------------------+-------------------------------------------------+
| ``AMPI_NUM_PROCESSES`` | Number of OS processes in the application       |
+------------------------+-------------------------------------------------+
| ``AMPI_MY_HOME_WTH``   | Home worker thread of the rank.                 |
+------------------------+-------------------------------------------------+
```

Comment (matthiasdiener) [reply doc/ampi/04-extensions.rst:479]: Also, the text above refers to windows. Is this still correct?

**Review by matthiasdiener — COMMENTED** [review]
We might want to extend the description of

File: doc/ampi/04-extensions.rst (hunk 1 above)
Comment (matthiasdiener) [reply doc/ampi/04-extensions.rst:479]: Also, the text above refers to 'host', which seems incorrect.

File: doc/ampi/04-extensions.rst [hunk 2]
```diff
@@ -477,7 +477,8 @@ threads, processes, and hosts in the job. We define a worker thread to
...
-``AMPI_MY_PROCESS``, ``AMPI_NUM_WTHS``, and ``AMPI_NUM_PROCESSES``.
+``AMPI_MY_PROCESS``, ``AMPI_NUM_WTHS``, ``AMPI_NUM_PROCESSES``, and
+``AMPI_MY_HOME_WTH``.
```
Comment (matthiasdiener) [inline doc/ampi/04-extensions.rst:481]:
```suggestion
```

**Review by epmikida — APPROVED** [review]

**Review by matthiasdiener — APPROVED** [review]

**Review by epmikida — APPROVED** [review]

**Pull request merged**